script/Ciel/Day 13/13_01_02_CIEL13_6.txt

```diff
@@ -650,52 +782,59 @@ When I open my eyes, I see that the Dead are already reaching toward me.
 [sha:c3a36dc702431dc3112ca323b06c75da8cc281d6]{
 -- Page 153, Offset 35684.
 -- 地面に倒れこんだ俺に殺到する手、手、手。
-Their hands swarm upon me as I lay collapsed on the ground.
+From where I lie on the ground,
+ I can see their groping, grasping hands pour down like rain.
 }
 [sha:ff8e5750a0d10b3afcc61f42570b6c31a78a5a79]{
 -- Page 153, Offset 35685.
 -- 倒れたまま転がって鉄柵から離れる。
-I roll away from the iron fence without getting back on my feet.
+Without getting up, I roll away from the iron fence.
 }
 [sha:8f0c8936f7bb4dd8cd874f04ac8c669a5ad4e6d6]{
 -- Page 153, Offset 35686.
 -- ノエルの指示なのか、死者たちは鉄柵の中には入ってこない。
-The Dead aren't going past the fence, likely because Noel ordered them not to.
+Noel must've ordered the Dead to stay put, as they aren't crossing over the fence.
 }
 [sha:99a7f8708b2344e14c54634d66014c050f0ee0ed]{
 -- Page 153, Offset 35687.
 -- となれば、間髪入れずにノエル自身の追撃が来る―――！
-Which means that Noel's going to come after me without a moment's notice―――!
+Which means... Noel herself is probably hot on my tail―――!
 }
 [sha:2495980b660ff3a88e76ad1769a66f4173c6ec6b]{
 -- Page 154, Offset 35688.
 -- 「あ―――ああああ―――ああもう、最っっっ高！」
-"AH―――AAAAH―――Ah, how incredible!"
+"%{i}AH%{/i}――%{i}AAAAH%{/i}――%{i}Ah%{/i}, I can't get enough!"
```

Comment:
Probably want that dramatic pause in there. Maybe some more caps. She's really hamming it up.
>%{i}AH%{/i}――%{i}AAAAH%{/i}――%{i}Ah%{/i}, this is just... %{i}INCREDIBLE%{/i}!